xpdan/data_reduction.py

```diff
@@ -80,7 +80,7 @@ def _feature_gen(self, event):
         for key in self.data_fields:
             val = event['data'].get(key, None)
             if el is not None:
-                feature = "{}={}".format(key, val)
+                feature = "{}".format(val)
```

Comment:
thanks for merging it. I was working on the test and not aware of it. Should we still finish this test?